libraries/botbuilder-ai/lib/luisRecognizer.d.ts

```diff
@@ -5,8 +5,7 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { Intent, IntentRecognizer } from 'botbuilder';
-import LuisClient = require('botframework-luis');
+import { Recognizer, RecognizerResult } from 'botbuilder';
```

Comment:
contextId has been marked obsolete for a while since it was connected to action binding.  Do we want to support it moving forward in the V4 SDK?

Reply:
Sure - removing.